src/glossary.rst

```diff
@@ -464,6 +464,24 @@ associated trait function
 An :dt:`associated trait function` is an :t:`associated function` that appears
 within a :t:`trait`.
 
+.. _fls_47xtji9Pk8Lw:
+
+associated trait implementation item
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Yeah same here, I really think it is clearer to have trait-implementation associated item, at a glance.